cpp/benchmarks/common/generate_input.cu

```diff
@@ -175,18 +231,30 @@ struct random_value_fn<T, std::enable_if_t<cudf::is_fixed_point<T>()>> {
     : lower_bound{desc.lower_bound},
       upper_bound{desc.upper_bound},
       dist{make_distribution<rep>(desc.id, desc.lower_bound, desc.upper_bound)}
-  {
-  }
-
-  T operator()(std::mt19937& engine)
   {
     if (not scale.has_value()) {
       int const max_scale = std::numeric_limits<rep>::digits10;
-      auto scale_dist     = make_distribution<int>(distribution_id::NORMAL, -max_scale, max_scale);
-      scale = numeric::scale_type{std::max(std::min(scale_dist(engine), max_scale), -max_scale)};
+      std::binomial_distribution<int> scale_dist_normal(2 * max_scale, 0.5);
+      std::mt19937 engine;
+      scale = numeric::scale_type{scale_dist_normal(engine) - max_scale};
```

Comment:
The scale will always have same value because we use `std::mt19937{}`. Since we need this to change, but keep deterministic, I will move this code below and update code to use `thrust::minstd_rand& engine`.